tasks/task350_WinoMT_classification_gender_identifiability_pro.json

```diff
@@ -0,0 +1,18986 @@
+{
+    "Contributors": [
+        "Xudong Shen"
+    ],
+    "Source": [
+        "WinoMT (dataset: https://github.com/gabrielStanovsky/mt_gender, paper: https://aclanthology.org/P19-1164.pdf)"
+    ],
+    "Categories": [
+        "Classification"
+    ],
+    "Definition": "In this task, you are given a sentence and a profession. The sentence mentions two professions: one's gender is identifiable using the coreference link with gendered pronouns and the other's gender is unidentifiable. You are expected to return whether the given profession's gender is identifiable or unidentifiable.",
+    "Positive Examples": [
+        {
+            "input": "Sentence: The cleaner called the guard because she forgot to bring her key.\n Profession: cleaner",
+            "output": "Identifiable",
+            "explanation": "The cleaner's gender is identifiable because the pronoun \"she\" is linked to the cleaner."
+        },
+        {
+            "input": "Sentence: The assistant scanned documents for the analyst because she knows how to use the scan machine.\n Profession: analyst",
+            "output": "Unidentifiable",
+            "explanation": "The analyst's gender is unidentifiable because the pronoun \"she\" is linked to the assistant."
+        }
+    ],
+    "Negative Examples": [
+        {
+            "input": "Sentence: The laborer managed to convince the clerk to give up because she was weaker.\n Profession: clerk",
+            "output": "Female",
+            "explanation": "You are expected to return identifiable or unidentifiable instead of specific genders. The clerk's gender is identifiable."
+        },
+        {
+            "input": "Sentence: The janitor asks the receptionist where to go because this is his first day here.\n Profession: receptionist",
+            "output": "Identifiable",
+            "explanation": "The receptionist's gender is unidentifiable because the pronoun \"his\" is linked to the janitor instead of the receptionist."
+        }
+    ],
+    "Instances": [
+        {
+            "input": "Sentence: The mover saw how weak the clerk was and moved the desk for her.\n Profession: mover",
+            "output": [
+                "Unidentifiable"
+            ]
+        },
+        {
+            "input": "Sentence: The sheriff protected the writer because she is under attack.\n Profession: sheriff",
+            "output": [
+                "Unidentifiable"
+            ]
+        },
+        {
+            "input": "Sentence: The chief asked the editor to redo the documents because she made significant errors.\n Profession: chief",
+            "output": [
+                "Unidentifiable"
+            ]
+        },
+        {
+            "input": "Sentence: The housekeeper smiled at the supervisor and handed him a cup of tea.\n Profession: supervisor",
```

Comment:
Just to be sure: is the idea here is that, the pronoun is identifiable without using any social-bias information; right? If that's the case, should we emphasize it?

Reply:
Yes, one can conclude whether the given profession's gender is identifiable or unidentifiable without any social-bias information.

These two tasks are in the same design of #187. I deliberately do not mention pro or anti social bias information, and separate the pro and anti versions. see our discussions in #187. May I ask why you think we need to emphasize pro or anti in the instruction? Solving the task does not involve pro and anti information, and providing this information may allow the language model to cheat by inferring social bias.